test/meuse_comp.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Meuse R gstat compatibility" begin
+    """meuse_grid.csv constains (x, y) plus estimation/variance of log(zinc)
+    for global and local (25 samples)
+    """
+    datadir = joinpath(@__DIR__,"data")
```

Comment:
I think this variable is already defined in runtests.jl? If not, please define it there like GeoStatsBase.jl does.

Reply:
Let me check it. One think that I dislike of Julia is this "include" mechanism, but I can live with that.

Reply:
What exactly you don't like about the include mechanism? Here we are not including files, just joining the path of the current directory where the source file lives with the subdirectory "data".

Reply:
Because the include mechanism makes the source code not reusable (elegantly) as it relays on some other "parent" source code that includes it. In this particular case, I need to remember that datadir was defined in runtest.jl.

I understand that this include mechanism is broadly used in Julia, but I dislike it. Nevertheless, there are many things that programmers dislike about languages but I still used them :-)